rs/rosetta-api/icp/Cargo.toml

```diff
@@ -18,15 +18,15 @@ dfn_candid = { path = "../../rust_canisters/dfn_candid" }
 dfn_protobuf = { path = "../../rust_canisters/dfn_protobuf" }
 hex = { workspace = true }
 ic-agent = { workspace = true }
-ic-canister-client = { path = "../../canister_client" }
-ic-limits = { path = "../../limits" }
 ic-crypto-sha2 = { path = "../../crypto/sha2/" }
 ic-crypto-tree-hash = { path = "../../crypto/tree_hash" }
 ic-crypto-utils-threshold-sig-der = { path = "../../crypto/utils/threshold_sig_der" }
+ic-limits = { path = "../../limits" }
 ic-ledger-canister-blocks-synchronizer = { path = "ledger_canister_blocks_synchronizer" }
 ic-ledger-canister-core = { path = "../../ledger_suite/common/ledger_canister_core" }
 ic-ledger-core = { path = "../../ledger_suite/common/ledger_core" }
 ic-ledger-hash-of = { path = "../../../packages/ic-ledger-hash-of" }
+ic-read-state-response-parser = { path = "../../canister_client/read_state_response_parser" }
```

Comment:
nit: These dependencies aren't sorted lexicographically, but they weren't to begin with, so the situation is not made any worse. I suppose CI only checks `BUILD.bazel` files for this, and not the `Cargo.toml`s.